.circleci/test-deploy.yml

```diff
@@ -50,6 +50,9 @@ jobs:
 
 workflows:
   test-deploy:
+    when:
```

Comment:
How does this `test-deploy` pipeline get used?

Reply:
after `lint-pack` is done, it will trigger `test-deploy`, so it is part of the personal branch pipeline

Reply:
Can you describe the handshake between this and the main `config.yml` file, and that this is necessitated by circle.